Sources/libMySQLWrapper/libMySQLWrapper.c

```diff
@@ -0,0 +1,338 @@
+//
+//  libMySQLWrapper.c
+//  SwiftKuery
+//
+//  Created by Matthew Kilner on 16/08/2018.
+//
+
+/**
+ Copyright IBM Corporation 2018
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#include "include/libMySQLWrapper.h"
+
+int wrapper_mysql_library_init(int argc, char **argv, char **groups)
+{
+    return mysql_library_init(0, NULL, NULL);
+}
```

Comment:
Do we need to wrapper every mysql function?  Can we not just wrapper the few where a conversion needs to happen?